collector/utils.go

```diff
@@ -112,8 +115,23 @@ func GetMem(item, unit string) int64 {
 	return GetRawInt64(item, "") / outUnit
 }
 
+func GetCPU(item string) int {
+	units := map[string]float64{
+		"m": 1000,
+	}
+
+	for key, value := range units {
```

Comment:
Whats point of range for 1 unit ? Are other units missing ?

Reply:
Fixed.